### ISSUE TITLE
The author did not release the training code, our reproduced results are far behind the reported results.

### ISSUE THREAD

**Issue by jessicaaaio1993** [Jiahao-UTS/SLPT-master#11]
The author has always refused to release the key training code and turned a blind eye to related problems. We can only reproduce the training code by ourselves, and the result is far from the accuracy of the paper. We ask the author to respond positively to the question and explain why only the test code is publicly available.

| WFLW dataset |NME $\downarrow$  | FR 0.1 $\downarrow$ | AUC 0.1 $\uparrow$|
|  ----  | ----  |----  |----  |
|paper（HRNetW18C-lite） | 4.14 | 2.76 | 0.595|
|our reproduced | 4.67 | 4.09 | 0.581|

| 300W dataset NME $\downarrow$  | Common|  Challenging |  Fullset |
|  ----  | ----  |----  |----  |
|paper（HRNetW18C-lite） | 2.75 |4.90 |3.17|
|our reproduced | 2.95 | 5.27 | 3.50|

**Comment by Jiahao-UTS** [comment]
Thanks for your interest. We only used the training set. You can find the training log at https://drive.google.com/file/d/1ZWUGq_11RwWOVSvojJZ8ttNJkgyNSaaj/view?usp=sharing. If you have problem, I think you should discuss the problem with us firstly, rather than register a new account and publish such aggressive words. Anyway, I never refuse publish the training code. Because of some issues, the training code is planned to be released in 2023. If you cannot reproduce the results, I think you should check your code firstly. Thank you!

**Comment by jessicaaaio1993** [comment]
Hi, author! Nice to see your first reply to the issue related to the missing training code. Unfortunately, you're still trying to prevaricate the problem. Many people mentioned the problem with the training code but you never replied to them. Can you respond to the question directly? It has been nearly a year since the paper was accepted, what caused you to procrastinate until now and not release the training code? You said that it would be released in 2023, which is very strange. Can you give us a specific deadline? Once the training code is published and can achieve the performance in the paper, I will apologize to you and retract all my comments. Or I will consider this as academic misconduct behavior and report this to the conference committee.

**Comment by Jiahao-UTS** [comment]
I have no responsibility to explain the reason to you. However, the code is definitely released in this year. It is a part of my plan but not for your comment. An apologize from a new account is so cheap. Who care?

**Comment by jessicaaaio1993** [comment]
It seems that you will not take responsibility for your and all co-authors' academic reputation. I feel sorry for that.

**Comment by Jiahao-UTS** [comment]
If you want to be treated politely, you should know how to be polite. Many people sent me email and I am very glad to discuss with them about their problem. Why are you so aggressive? Anyway, I still thank for your interest and spending time to reproduce the code. The code is definitely released in this year, and I will give you a reply at that time.
